### ISSUE TITLE
Add Github Action workflow to run tests

### ISSUE THREAD

**Issue by tamuri** [UCL-ARC/shortlister#72]
Using [Github Actions](https://docs.github.com/en/actions) write a workflow to run the tests on pull requests and new merges into main branch.

**Comment by shiyingwucl** [comment]
Is there a way to test if the workflow correctly runs after opening a pull request/merging without manually doing so?

**Comment by tamuri** [comment]
Yes, you set up a Github Action as described here: https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/triggering-a-workflow

Working through the documentation will help to understand Github Actions

**Comment by shiyingwucl** [comment]
Sorry, I might have been unclear with my question. I meant if I had a GitHub workflow that is suppose to run when a merge happens, could I see if it runs/or doesn't run without actually merging into the branch?

**Comment by tamuri** [comment]
Ah, no - you have to trust the configuration 😄 . I can see it worked for the pull request, which is a good indication it will work when you merge.
